### PR TITLE
Fix/try except modules

### LINKED ISSUE #74: Try/except should be removed for loading modules
Hi @santiagoinfantinom! I have noticed that there are still some try/excepts when loading the modules, eg. [here](https://github.com/rl-institut/offgridders/blob/dev/src/G0_oemof_simulate.py#L21). Is there a way to get rid of those? In the MVS, we do not have these exceptions at all anymore.

### PR DESCRIPTION
**Pull request by santiagoinfantinom**
Fix #74 
PR erasing try/except requests at importation

### CODE REVIEW

**Comment by smartie2076** [comment]
@santiagoinfantinom same here - I did not see that I can review this as I am not assigned to be a reviewer.

File: CHANGELOG.md [hunk 1]
```diff
@@ -14,7 +14,8 @@ Here is a template for new release sections
 ### Changed
 -
 ### Removed
--
+- Try/Except clauses at module import
```
Comment (smartie2076) [inline CHANGELOG.md:17]: Please add after this line the PR that the change is connected to. That way, in the future, one can easily find what has been changed.

Comment (santiagoinfantinom) [reply CHANGELOG.md]: ddaa0ec5, does this fix it? Can this PR be merged?

File: CHANGELOG.md [hunk 2]
```diff
@@ -14,7 +14,8 @@ Here is a template for new release sections
 ### Changed
 -
 ### Removed
--
+- Try/Except clauses at module import (Fixed PR #103)
```
Comment (smartie2076) [inline CHANGELOG.md:17]:
```suggestion
- Try/Except clauses at module import (#103)
```

**Comment by smartie2076** [comment]
@santiagoinfantinom you can merge this now :)

**Pull request merged**